README.md

```diff
@@ -248,3 +248,7 @@ A sample run:
     Completed all reads
 
 Like "many-clients", it's just a load test.
+
+## See also
+
+(Siberite)[https://github.com/bogdanovich/siberite] - Kestrel rewritten with Go.
```

Comment:
I think you reversed the link order. The link text goes in square brackets, the link itself goes in parens.